apps/sms/locales/sms.en-US.properties

```diff
@@ -308,6 +349,19 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-notice-label.ariaLabel           = Late arrival
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = {{ lateness }}mth late
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[two]    = {{ lateness }}mths late
+late-arrival-notice-in-months[few]    = {{ lateness }}mths late
+late-arrival-notice-in-months[many]   = {{ lateness }}mths late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
```

Comment:
I think "mth" is quite confusing, several locales won't have such an abbreviation and will end up using the full "month" word, taking up a lot of space. I agree with gandalf, something like "Late arrival: XXX" should be easier to manage on the localization side.